### PR TITLE
Set Env Var following the pattern of configuration

### PR DESCRIPTION
**Pull request by FrauleinGela**
I added the Env Var to Flags so I use the Prefix from the serverConfig.
Also, I do not know what you mean this line.
> but anyway the default prefix should be no prefix, so it doesn't break our current deploy.

### CODE REVIEW

**Comment by FrauleinGela** [comment]
I explain the process I followed:
go get -u github.com/MEDIGO/laika/client
make install
make migrate
make watch --- throws the following error:
`Watching source code...

laika@0.2.0 watch /Users/ngela/go/src/github.com/MEDIGO/laika
npm run build -- --watch
laika@0.2.0 build /Users/ngela/go/src/github.com/MEDIGO/laika
webpack --inline --colors --progress --display-error-details --display-cached "--watch"
git/ivp/rer 19:06:40 ♥[INFO] ▶ build took 2.24(s)
git/ivp/rer 19:06:40 ♥[INFO] ▶ rerun started. watching for changes...
{"level":"info","msg":"Starting server on port 8000","time":"2016-12-19T19:06:40+01:00"} 90% optimize assets
/Users/ngela/go/src/github.com/MEDIGO/laika/node_modules/typescript/lib/typescript.js:3177
throw new Error("Debug Failure. False expression: " + (message || "") + verboseDebugString);
^
Error: Debug Failure. False expression: File has unknown extension.
at Object.assert (/Users/ngela/go/src/github.com/MEDIGO/laika/node_modules/typescript/lib/typescript.js:3177:23)
at Object.fail (/Users/ngela/go/src/github.com/MEDIGO/laika/node_modules/typescript/lib/typescript.js:3182:19)
...
`

**Comment by christophstrasen** [comment]
Hey there. I got the same problem with typescript.js:3177. I solved it with bumping the typscript version in the package.json

Do you want to try that?

**Comment by apatinoyu** [comment]
yay, cool.
Will try today after 18h , i let you know

**Comment by FrauleinGela** [comment]
yay, cool.
Will try today after 18h , I let you know.

File: main.go [hunk 1]
```diff
@@ -17,10 +17,11 @@ import (
 func init() {
 	log.SetFormatter(&log.JSONFormatter{})
 }
-
+//TODO set Prefix by echo Router library?
 func main() {
 	app := cli.NewApp()
 	app.Author = "MEDIGO GmbH"
+	os.Setenv("PREFIX_ROUTE", "/laika") //Added a prefix as an environment variable
```
Comment (dbrabera) [inline main.go]: Remove this. You already added a prefix on the config below.

File: main.go [hunk 2]
```diff
@@ -93,6 +94,12 @@ func main() {
 			Usage:  "Slack webhook URL",
 			EnvVar: "LAIKA_SLACK_WEBHOOK_URL",
 		},
+		cli.StringFlag{
+			Name: "route-prefix",
+			Usage: "Route prefix",
+			EnvVar: "LAIKA_ROUTE_PREFIX",
+			Value: "/laika",
```
Comment (dbrabera) [inline main.go]: The default value should be _the empty string_, because we don't always need to prefix the routes

**Comment by dbrabera** [comment]
I added a couple of comments. We want the default prefix (when the LAIKA_PREFIX variable is not set) to be the empty string. That way we can opt to have a prefix or none at all.

**Comment by FrauleinGela** [comment]
@christophstrasen ,  I tried to update the package.json to typescript version: 2.0.2 and ^2.1.4 (the latest), however, continue throwing the same error.
let me know in case you fixed it by updating to an older version. otherwise, I will try to go deeper with the config issue. 
Regards.

**Pull request closed without merging**